lib/client.js

```diff
@@ -228,14 +228,29 @@ extend(Raven.prototype, {
       so we only parse a `req` property if the `request` property is absent/empty (and hence we won't clobber)
       parseUser returns a partial kwargs object with a `request` property and possibly a `user` property
     */
+    var initialRequest = utils.prepareInitialRequest(
+      this._globalContext.request,
```

Comment:
would you mind adding some comments here? I'm having trouble understanding why it is needed to use both `prepareInitialRequest` and `extend` on these `request` & `req` objects, instead of just `prepareInitialRequest.

Reply:
This was just me missing it. Fixed :)